kuma/core/templatetags/jinja_helpers.py

```diff
@@ -92,11 +92,6 @@ def entity_decode(str):
     return htmlparser.unescape(str)
 
 
-@library.global_function
-def page_title(title):
```

Comment:
Bold! 😮 

I would have changed this function to not add `` | MDN`` to titles for the experiment, but this way (removing it here and from all the templates) works as well. My best guess is that it won't come back after we remove it.